hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1248 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d = salobj.Domain()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script = salobj.Controller(\"Script\", index=1)\n",
+    "rot = salobj.Remote(d, \"MTRotator\")\n",
+    "mtptg = salobj.Remote(d, \"MTPtg\")\n",
+    "mtm = salobj.Remote(d, \"MTMount\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.gather(rot.start_task,\n",
+    "                     mtptg.start_task,\n",
+    "                     script.start_task,\n",
+    "                     mtm.start_task)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Enable the Pointing component:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(1.)\n",
+    "await salobj.set_summary_state(mtptg, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Rotator-Mount start-up sequence:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await rot.cmd_start.set_start(timeout=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.DISABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(rot, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Rotator state machine commands: (only use when needed)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await rot.cmd_clearError.start(timeout=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(rot, salobj.State.DISABLED)                             "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await rot.cmd_start.set_start(timeout=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await rot.cmd_enable.set_start(timeout=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(rot, salobj.State.STANDBY)        "
```

Comment:
I suggest not enabling the rotator unless the MTMount is enabled first. It is much safer. That way the MTMount has a chance to make the CCW track the rotator. 

I realize that for certain very specialized tests you may want to turn off CCW following. But this section seems to be some sort of general preparation.

I bring this up here because I see you send MMTMount to DISABLED shortly above, but never all the way to ENABLED. There is a comment *below* (at line 359) that describes the correct sequence. So it feels like things are out of order. That explanation should be up here and the sequence here should follow the description.

Reply:
Yes, starting up and shutting down the system properly is part of a bigger effort to create a setup notebook that will replace the great majority of state-machine-related commands in the notebook mentioned here. For the moment left alone here.